File: programming-basics-javascript/02b-conditional-statements-exercise/08-lunch-break.js
function lunchBreak(input) {
  let seriesName = input[0];
  let episodeDuration = Number(input[1]);
  let breakDuration = Number(input[2]);

  lunchTime = breakDuration * 0.125;
  restTime = breakDuration * 0.25;
  usedTime = lunchTime + restTime;
  leftTime = breakDuration - usedTime;

  absTime = Math.abs(leftTime - episodeDuration);
  absTime = Math.ceil(absTime);

  if (leftTime >= episodeDuration) {
    console.log(
      `You have enough time to watch ${seriesName} and left with ${absTime} minutes free time.`
    );
  } else {
    console.log(
      `You don't have enough time to watch ${seriesName}, you need ${absTime} more minutes.`
    );
  }
}

File: b1-programming-basics-python/02b-conditional-statements-exercise/08_lunch_break.py
import math

series_name = input()
series_time = int(input())
rest_time = int(input())

lunch_time = rest_time * 0.125
down_time = rest_time * 0.25
time_used = lunch_time + down_time
time_left = rest_time - time_used

time_abs = abs(time_left - series_time)
time_round = math.ceil(time_abs)

if time_left > series_time:
    print(
        f"You have enough time to watch {series_name} and left with {time_round} minutes free time."
    )

else:
    print(
        f"You don't have enough time to watch {series_name}, you need {time_round} more minutes."
    )
